.husky/pre-push

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+yarn test
```

Comment:
Sneaking this in =) 

(I've found this is the least controversial of all the hooks as it runs infrequently and fairly quickly, and I've found it extremely helpful).